packages/edit-site/src/components/sidebar-navigation-screen-global-styles/index.js

```diff
@@ -198,8 +160,8 @@ export default function SidebarNavigationScreenGlobalStyles() {
 				content={ <SidebarNavigationScreenGlobalStylesContent /> }
 				footer={
 					shouldShowGlobalStylesFooter && (
-						<SidebarNavigationScreenGlobalStylesFooter
-							modifiedDateTime={ modifiedDateTime }
+						<SidebarNavigationScreenDetailsFooter
+							record={ revisions?.[ 0 ] }
```

Comment:
This is still in progress, but before I go further do you see anything dodgy about what I'm doing here @andrewserong 😄 

Things look the same, but I'm concerned I'm locking the global styles screen into the generic footer.

Reply:
Thanks for the ping! From a quick glance, I think this looks like a good direction! From my understanding the goal is for us to achieve consistency between a bunch of these screens, and the purpose of the footer area last modified section is much the same between updating a page or updating styles, it's just that we're pulling data from a different place and the `onClick` will take us somewhere different, but what's communicated to a user will be much the same 👍